engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -1202,6 +1216,7 @@ namespace dmGameObject
 
         bool success = CreateComponents(collection, instance);
         if (!success) {
+            dmLogError("Could not create components.");
```

Comment:
If we don't have the reason here, then the reason should be printed in the CreateComponents() perhaps?
I don't feel this error message is super helpful tbh (but still adds to engine size)